src/ray/pubsub/publisher.cc

```diff
@@ -98,8 +103,23 @@ const absl::flat_hash_map<SubscriberID, SubscriberState *> &EntityState::Subscri
   return subscribers_;
 }
 
-SubscriptionIndex::SubscriptionIndex(rpc::ChannelType channel_type)
-    : channel_type_(channel_type), subscribers_to_all_(CreateEntityState()) {}
+SubscriptionIndex::SubscriptionIndex(rpc::ChannelType channel_type,
+                                     int64_t max_message_size_bytes)
+    : max_message_size_bytes_(max_message_size_bytes),
+      channel_type_(channel_type),
+      subscribers_to_all_(CreateEntityState()) {}
```

Comment:
While you're at it, can you please make this one static (and pass params explicitly)